CHANGELOG.asciidoc

```diff
@@ -124,6 +124,8 @@ https://github.com/elastic/beats/compare/v6.0.1...v6.1.0[View commits]
 - Change `MySQL active connections` visualization title to `MySQL total connections`. {issue}4812[4812]
 - Fix `ProcState` on Linux and FreeBSD when process names contain parentheses. {pull}5775[5775]
 - Fix incorrect `Mem.Used` calculation under linux. {pull}5775[5775]
+- Fix mongodb session consistency mode to allow command execution on secondary nodes. {issue}4689[4689]
```

Comment:
I don't think this line was intentional. Merge issue?

Reply:
yep, one of the famous rebase issues :-( Fixed.